consensus/src/step_votes_reg.rs

```diff
@@ -6,92 +6,125 @@
 
 use crate::commons::{IterCounter, RoundUpdate, StepName};
 use crate::config::CONSENSUS_MAX_ITER;
-use node_data::ledger::to_str;
 use node_data::ledger::StepVotes;
+use node_data::ledger::{to_str, Certificate};
 use node_data::message::{payload, Message, Topics};
 use std::fmt;
 use std::sync::Arc;
 use tokio::sync::Mutex;
-use tracing::{debug, error};
+use tracing::debug;
 
 pub(crate) enum SvType {
     FirstReduction,
     SecondReduction,
 }
 
-#[derive(Default, Copy, Clone)]
-struct SvEntry {
-    // represents candidate block hash
+#[derive(Default, Clone, Copy)]
+struct AgreementInfo {
```

Comment:
Not sure I'd call it AgreementInfo, as it sounds like an Agreement was reached.
In fact this is more a ReductionInfo since it collects known Reduction votes.

Anyway, feel free to leave it. We'll probably address this when introducing Attestations.

Reply:
You're right. Initially, `AgreementInfo` was designed to keep certificates with either NIL or Full quorum. The name now is a bit misleading. I'd suggest to use `CertificateInfo`  instead.

Reply:
While Certificate is coherent with the current naming system, with the introduction of `Attestation`s, a `Certificate` will only be the one certifying the previous block (and including both Agreement and Failed iteration votes). 
What about `ConsensusInfo`? Too generic?